frontend/src/metabase-lib/v1/Question.ts

```diff
@@ -778,22 +771,17 @@ class Question {
     }
 
     const newQuery = this.parameters().reduce((query, parameter) => {
-      if (isFilterParameter(parameter)) {
-        const stageIndex =
-          isDimensionTarget(parameter.target) && !isComposed
-            ? getParameterDimensionTargetStageIndex(parameter.target)
-            : -1;
-        return applyFilterParameter(query, stageIndex, parameter);
-      } else if (isTemporalUnitParameter(parameter)) {
-        const stageIndex =
-          isDimensionTarget(parameter.target) && !isComposed
-            ? getParameterDimensionTargetStageIndex(parameter.target)
-            : -1;
-
-        return applyTemporalUnitParameter(query, stageIndex, parameter);
-      } else {
-        return query;
-      }
+      const stageIndex =
+        isDimensionTarget(parameter.target) && !isComposed
+          ? getParameterDimensionTargetStageIndex(parameter.target)
+          : -1;
+      return applyParameter(
```

Comment:
There are separate `type`, `target`, and `value` because this combining is not uniform across all parameter objects in our app. `target` and `value` are included into `parameters` only here.